7-differential-expression.rst

```diff
@@ -1,4 +1,20 @@
 7. Extracting differentially expressed genes with edgeR
 =======================================================
 
-edgeR.
+Run this `edgeR <https://bioconductor.org/packages/release/bioc/html/edgeR.html>`__ script (`nema.salmon.R
+<https://raw.githubusercontent.com/dib-lab/eel-pond/DE/nema.salmon.R>`__)
+that loads all this in and calculates a few plots --
```

Comment:
Do we want to provide any more context? Could be as simple as *for details, see the source code and the edgeR documentation* or something like that.